dbmongo/go.mod

```diff
@@ -4,22 +4,17 @@ go 1.14
 
 require (
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
-	github.com/appleboy/gin-jwt v2.5.0+incompatible
```

Comment:
Ah oui, c'était le traitement des jwt made in gin, avec keycloak, pas besoin de tout ça, les fonctions qui faisaient ça ont été retirées lors du nettoyage précédent.